tools/migrator/src/main/java/org/wso2/carbon/ei/migration/MigrationClient.java

```diff
@@ -0,0 +1,6 @@
+package org.wso2.carbon.ei.migration;
```

Comment:
Add Licence headers and doc comments.

Reply:
I have added the licence headers and doc comments.